hsts.js

```diff
@@ -0,0 +1,15 @@
+function handler(event) {
+  var response = event.response;
+  var headers = response.headers;
+
+  //Set new headers
+  headers['strict-transport-security'] = {'value': 'max-age=31536000'};
+  // the following are not used as of 2018-12-26, but can be commented back in when appropriate
+  //headers['content-security-policy'] = {'value': "default-src 'none'; img-src 'self'; script-src 'self'; style-src 'self'; object-src 'none'"};
+  headers['x-content-type-options'] = {'value': 'nosniff'};
```

Comment:
I think we only want strict-transport-security.  Some of these other configs can cause problems (we _definitely_ had some weird stuff going on with ELI)

Reply:
Made the necessary changes and re-deployed.